app/src/main/java/com/postpc/tenq/ui/activities/MainActivity.java

```diff
@@ -26,15 +32,53 @@
 public class MainActivity extends TenQActivity {
 
     private static final int REQUEST_CODE = 1337;
+    private static RecorderService recorder;
+
+//    @Override
+//    protected void onCreate(Bundle savedInstanceState) {
+//        super.onCreate(savedInstanceState);
+//        setContentView(R.layout.activity_main);
+//
+//        startSpotifyAuthFlow();
+//
+//    }
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
```

Comment:
this looks like you're overriding what's in the main activity for your own testing... Please don't do that :)

Reply:
Actually, I added the recording initialization to this part. I just wasn't sure I placed it in the right place so I left this section in a comment. The next section is starting the same activities but controlling the recording too. So should I stick with my change?

Reply:
You should keep the MainActivity that's in master as is.
The recorder should only be initialized when music is playing (and the "sound awareness" feature is turned on).
If you're using the main activity to test your code it's ok, but don't push with your changes

Reply:
You should keep the MainActivity that's in master as is.
The recorder should only be initialized when music is playing (and the "sound awareness" feature is turned on).
If you're using the main activity to test your code it's ok, but don't push with your changes

Reply:
We have to ask the users for permission to use their microphone, so where should I put this step?